components/automate-ui/src/app/components/admin-sidebar/admin-sidebar.component.html

```diff
@@ -26,8 +26,8 @@
       <chef-sidebar-entry route="/settings/tokens" icon="vpn_key">API Tokens</chef-sidebar-entry>
     </app-authorized>
 
-    <div *ngIf="(iamMajorVersion$ | async) !== 'v1'">
-      <div class="group" *ngIf="policies.visible || roles.visible || projects.visible">Access Management</div>
+    <div *ngIf="(iamMajorVersion$ | async) === 'v2'">
```

Comment:
I could see the case for keeping this as a `!== v1` (if we bump to a 3?) . . . I just found the positive `===` so much easier to read 🤷‍♂